net-vpn/candy/candy-5.10.3.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake systemd readme.gentoo-r1
+
+DESCRIPTION="A reliable, low-latency, and anti-censorship virtual private network"
+HOMEPAGE="https://github.com/lanthora/candy"
+SRC_URI="https://github.com/lanthora/candy/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
I thought that was only a "convention" for solving filename conflicts when a previous ${P}.tar.gz was downloaded from some other location and ended up in distfiles caches and GENTOO_MIRRORS.

Reply:
yes, as I said it's a convention, rather than a soft policy..

but I do believe, it has some benefits to show people this tarball was downloaded from github's release tag and renamed to ${P} specific, also it will avoid future collision if maintainer find some more official location to release his code

Reply:
IIRC it was due to pypi release tarballs and github release tarballs having different contents (the pypi tarball didn't often come with tests shipped).